### PR TITLE
[ADD] stock_bin_location

### PR DESCRIPTION
**Pull request by jantoniorivera97**

### CODE REVIEW

File: stock_bin_location/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+Purchase App:
+
+Click on 'Purchase' menu and then click on 'Requests for Quotation'.
```
Comment (patrickrwilson) [inline stock_bin_location/README.rst]: I don't think the purchase app info here is needed as this module doesn't depend on the purchase. So, only the inventory section is needed.

File: stock_bin_location/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2021 Open Source Integrators
```
Comment (patrickrwilson) [inline stock_bin_location/__manifest__.py]: Small detail, change year to 2022 where applicable.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Looks good except there are a few items to fix.
It looks like there are modifications to modules other than the new stock_bin_location module you are trying to add. This PR should only contain the module you are adding. Can you fix that please. 
Once the above is fixed, if you have muliple commits, you will want to squash them into a single commit. It helps the review process as well as cleans up history. This is a good reference if you aren't familiar with squashing. https://www.git-tower.com/learn/git/faq/git-squash

File: stock_bin_location/README.rst (hunk 1 above)
Comment (blitzkrieg-ar) [reply stock_bin_location/README.rst]: I think it's done. I've changed the year to 2022, edited the Usage in the README file and uploaded only this module modifications. Thanks for sharing the squashing article.

**Comment by patrickrwilson** [comment]
@jantoniorivera97 looks like this is close, pre-commit is complaining about two lines in the readme: 
************* Module stock_bin_location
stock_bin_location/README.rst:3: [E7901(rst-syntax-error), ]  Enumerated list start value not ordinal-1: "2" (ordinal 2)
stock_bin_location/README.rst:3: [E7901(rst-syntax-error), ]  Enumerated list start value not ordinal-1: "3" (ordinal 3)

**Pull request closed without merging**